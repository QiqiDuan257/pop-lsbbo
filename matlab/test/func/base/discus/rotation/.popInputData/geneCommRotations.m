clear; clc;

% nohup matlab -nodesktop -nodisplay -nosplash <geneCommRotations.m >geneCommRotations.out 2>&1 &

%%
popGeneCommRotations(20190907, popGeneCommFuncDims());
% popGeneCommRotations(201909067, 3 : 7);
