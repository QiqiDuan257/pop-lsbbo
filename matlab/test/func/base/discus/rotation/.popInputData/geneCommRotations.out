
                            < M A T L A B (R) >
                  Copyright 1984-2016 The MathWorks, Inc.
                   R2016b (9.1.0.441655) 64-bit (glnxa64)
                             September 7, 2016

#############################################################################
#             $$$ Enjoy the <pop-lsbbo> pure-MATLAB Library $$$             #
#                        --- version: 0.00.000.1 ---                        #
#############################################################################
     2: runtime 2.47e-01.
    10: runtime 1.12e-02.
   100: runtime 3.45e-02.
  1000: runtime 2.82e+00.
  2000: runtime 1.78e+01.
  4000: runtime 3.28e+02.
  6000: runtime 8.90e+02.
  8000: runtime 1.78e+03.
 10000: runtime 3.18e+03.
Total runtime: 6.20256e+03.
